website/components/blog/PostPreview.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react"
+import Link from 'next/link'
+
+export interface Text {
+    type: string 
+    text: string
+    spans: Array<any>
+}
+
+interface MainImg {
+    url: string
+    dimensions: {
+        width: number
+        height: number
+    }
+    alt: string | null
+    copyright: string | null
+}
+
+export interface PostData {
+    title?: Array<Text>
+    main_image?: MainImg
+    teaser?: Array<Text>
+}
+
+export interface Post {
+    data: PostData
+    tags: Array<string>
+    uid: string
+    id: string
+    index: number
+}
+
+interface Props {
+    post: Post
+    columnsOccupied?: number
+    isLargeTitle?: boolean
+}
+
+const PostPreview: React.FC<Props> = ({ post, columnsOccupied = 12, isLargeTitle = true}) => {
```

Comment:
no space between true and `})` - can you run prettier via `npm run lint:fix` command?